app/controllers/application_controller.rb

```diff
@@ -157,6 +158,9 @@ def download_summary_pdf
     @embedded = true
     @showlinks = false
 
+    # encode images and embed in HTML that is sent to Prince
+    @base64_encode_images = true
```

Comment:
Additionally, I don't really like this instance variable hanging out here.  I think it might be better if the ImageEncodeHelper has an attr_accessor for this value (it already has the predicate getter), and then either the default is set in that getter on first access, or the value is set here through the setter `self.base64_encode_images = true`